### PR TITLE
[faucet] task: reconfigure .github

### PR DESCRIPTION
**Pull request by AnthonyLaw**
config `.github` and `lint` path.

### CODE REVIEW

**Review by Jaguar0625 — APPROVED** [review]

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]
please update main README

**Comment by AnthonyLaw** [comment]
> please update main README

I update the readme table and pre-build the URL following the previous link

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -80,3 +80,34 @@ updates:
...
+      prefix: '[dependency]'
+
+  - package-ecosystem: npm
+    directory: /faucet/frontend
+    schedule:
+      interval: weekly
+      day: sunday
+    target-branch: dev
+    labels: [Faucet Frontend]
+    versioning-strategy: increase
```
Comment (Wayonb) [inline .github/dependabot.yaml:111]: add this ``versioning-strategy`` to backend and authenticator also

File: README.md [hunk 2]
```diff
@@ -50,3 +53,21 @@ Detailed version can be seen on [codecov.io][product-cov-link].
...
+
+[faucet-backend-job]: https://jenkins.symboldev.com/blue/organizations/jenkins/Symbol%2Fgenerated%2Fproduct%2Ffaucet-backend/activity?branch=dev
+[faucet-backend-lint]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fproduct%2Ffaucet-backend%2Fdev%2F&config=faucet-backend-lint
+[faucet-backend-test]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fproduct%2Ffaucet-backend%2Fdev%2F&config=faucet-backend-test
+[faucet-backend-cov]: https://codecov.io/gh/symbol/product/branch/dev/graph/badge.svg?token=SSYYBMK0M7&flag=faucet-backend
+[faucet-backend-cov-link]: https://codecov.io/gh/symbol/product/tree/dev/faucet/backend
+
+[faucet-frontend-job]: https://jenkins.symboldev.com/blue/organizations/jenkins/Symbol%2Fgenerated%2Fproduct%2Ffaucet-frontend/activity?branch=dev
+[faucet-frontend-lint]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fproduct%2Ffaucet-frontend%2Fdev%2F&config=faucet-frontend-lint
+[faucet-frontend-test]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fproduct%2Ffaucet-frontend%2Fdev%2F&config=faucet-frontend-test
```
Comment (Wayonb) [inline README.md:71]: missing the build section for frontend?

**Review by Wayonb — CHANGES_REQUESTED** [review]

**Review by Jaguar0625 — COMMENTED** [review]
i missed this first time, but please alphabetize everything

so faucet should appear after explorer, not at end of files

**Comment by Jaguar0625** [comment]
> i missed this first time, but please alphabetize everything
> 
> so faucet should appear after explorer, not at end of files

Seems like you still need to reorder in some files - (buildConfiguration, codecov, etc)

File: .github/dependabot.yaml [hunk 3]
```diff
@@ -80,3 +80,36 @@ updates:
     target-branch: dev
```
Comment (Jaguar0625) [inline .github/dependabot.yaml:80]: alphabetize this file

File: .gitlintmodules [hunk 4]
```diff
@@ -1 +1 @@
-optin/puller|optin/reporting|optin/reporting/client|tools/vanity|monorepo|explorer/nodewatch|lightapi/python
+optin/puller|optin/reporting|optin/reporting/client|tools/vanity|monorepo|explorer/nodewatch|lightapi/python|faucet/authenticator|faucet/backend|faucet/frontend
```
Comment (Jaguar0625) [inline .gitlintmodules:1]: please alphabetize

File: README.md [hunk 5]
```diff
@@ -10,6 +10,9 @@ It includes our optin manager.
 | [@optin/puller](optin/puller) | [![lint][optin-puller-lint]][optin-puller-job] | | [![test][optin-puller-test]][optin-puller-job]| [![][optin-puller-cov]][optin-puller-cov-link] |
 | [@optin/reporting](optin/reporting) | [![lint][optin-reporting-lint]][optin-reporting-job] | [![build][optin-reporting-build]][optin-reporting-job] | [![test][optin-reporting-test]][optin-reporting-job]| [![][optin-reporting-cov]][optin-reporting-cov-link] |
 | [@tools/vanity](tools/vanity) | [![lint][tools-vanity-lint]][tools-vanity-job] | | [![test][tools-vanity-test]][tools-vanity-job]| [![][tools-vanity-cov]][tools-vanity-cov-link] |
+| [@faucet/authenticator](faucet/authenticator) | [![lint][faucet-authenticator-lint]][faucet-authenticator-job] | | [![test][faucet-authenticator-test]][faucet-authenticator-job]| [![][faucet-authenticator-cov]][faucet-authenticator-cov-link] |
```
Comment (Jaguar0625) [inline README.md:13]: please alphabetize

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**